### PR TITLE
 #issue-795 convert column charts to VerticalBarChart class

### PR DESCRIPTION
**Pull request by fozy81**
This fixes issue https://github.com/ember-learn/ember-website/issues/795

### CODE REVIEW

**Comment by fozy81** [comment]
Hi @ijlee2 , please review these changes.

File: app/controllers/ember-community-survey-2018.js [hunk 1]
```diff
@@ -471,180 +409,135 @@ const emberenoEmployment = new HorizontalBarChart({
...
-    xAxis: { type: 'category' },
-    yAxis: { title: { text: 'Percent' } },
+}).highchartsOptions;
+
+const demographicsRegion = new VerticalBarChart({
+  chart: {
+    categories: [
+      'North America',
+      'Europe	',
+      'Asia	',
```
Comment (ijlee2) [inline app/controllers/ember-community-survey-2018.js:522]:
```suggestion
      'Europe',
      'Asia',
```

**Review by ijlee2 — APPROVED** [review]
Wow, that was very fast. Thanks again!

**Comment by fozy81** [comment]
I've got into a rhythm! Thanks for the review.

**Pull request merged**